tests/Jobs/Integration/QueryserviceNamespaceJobTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Tests\Jobs\Integration;
+
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Tests\TestCase;
+use App\Jobs\ProvisionQueryserviceNamespaceJob;
+use App\QueryserviceNamespace;
+use App\Jobs\DeleteQueryserviceNamespaceJob;
+use App\User;
+use App\Wiki;
+use App\WikiManager;
+use Illuminate\Support\Facades\DB;
+use Carbon\Carbon;
+use Illuminate\Contracts\Queue\Job;
+
+/**
+ * This is only meant to run when services is started with 
+ * additional services from docker-compose.integration.yml
+ * 
+ * Example: docker-compose exec -e RUN_PHPUNIT_INTEGRATION_TEST=1 -T api vendor/bin/phpunit tests/Jobs/Integration/QueryserviceNamespaceJobTest.php
+ */
+class QueryserviceNamespaceJobTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    public function testIntegrationCreate()
+    {
+        if ( !getenv('RUN_PHPUNIT_INTEGRATION_TEST') ) {
```

Comment:
Might be nicer / cleaner to have the env var check in setUp or setUpBeforeClass to keep it seperate from the actuall test case / cases if more are added

Reply:
good idea